health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/attendanceManagementTable.js

```
@@ -1,92 +1,175 @@
-    // }
-    props?.onRowSelect(event);
-  };
+    if (value?.target) {
+      value = value?.target?.value;
+    }
 
-  const handlePerRowsChange = async (currentRowsPerPage, currentPage) => {
-    props?.handlePerRowsChange(currentRowsPerPage, currentPage);
-  };
+    // Remove leading zeros from the value
+    value = String(value).replace(/^0+/, "");
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)